### ISSUE TITLE
Feature Request: Admin mailing of new user registrations waiting in approval queue.

### ISSUE THREAD

**Issue by merpdotcom** [collective/collective.emailconfirmationregistration#5]
Email notification when there is a registration request waiting in the queue. With juggling all those sites, it is very time consuming to keep checking each one for anyone in the queue. When that "enable" button for admin approval of account notification is enabled, some feature that sends email to the admin that an account is now waiting in the queue as notification to the admin(s) would be very helpful.

**Comment by rileydog** [comment]
Hi,  When testing this on P5, all went well for installation, etc.  When trying to register, I got the generic "error submitting form".  This is the error from my site:
Traceback (innermost last):
  Module ZPublisher.Publish, line 138, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 48, in call_object
  Module z3c.form.form, line 218, in **call**
  Module plone.z3cform.fieldsets.extensible, line 59, in update
  Module plone.z3cform.patch, line 30, in GroupForm_update
  Module z3c.form.group, line 145, in update
  Module plone.app.z3cform.csrf, line 21, in execute
  Module z3c.form.action, line 98, in execute
  Module z3c.form.button, line 315, in **call**
  Module z3c.form.button, line 170, in **call**
  Module collective.emailconfirmationregistration.browser, line 216, in action_verify
  Module collective.emailconfirmationregistration.browser, line 161, in send_mail
AttributeError: email_from_address

(Note:  I did confirm that my site has a from  email for the site (not sure if that is the email referred to.

Any help would be appreciated because for P5 non of the captcha programs work, and on a simple test site of mine, I got a dozen spam registrations in a day
